### PR TITLE
docs: move history and collections features to dedicated page

### PR DESCRIPTION
**Pull request by sennierer**
features that are shipped with APIS, but need to be activated and/or configured are moved to a dedicated "Additional Features" page

### CODE REVIEW

File: docs/source/features.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+===================
+
+APIS is designed to be extensible and customizable. This section describes some 
+additional features that are shipped with APIS and how to use them.
+
+History plugin
+--------------
+
+The :py:mod:`apis_core.history` module provides versioning 
+to the APIS framework. It is based on the `django-simple-history`_
```
Comment (b1rger) [inline docs/source/features.rst:11]:
```suggestion
for APIS. It is based on the `django-simple-history`_
```

File: docs/source/features.rst [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+
+
+API endpoint
+^^^^^^^^^^^^
+
+The :class:`apis_core.history.api_views.GenericHistoryLog` class is a viewset
+that provides a REST API endpoint for the version history of a model. It can be
+used to retrieve the version history of a model instance.
+The viewset can be accessed under `/apis/api/history/entity/edit_log/`. It takes
```
Comment (b1rger) [inline docs/source/features.rst:40]: I don't find `edit_log` anywhere in the codebase, I guess that has changed?

**Review by b1rger — CHANGES_REQUESTED** [review]

File: docs/source/features.rst (hunk 2 above)
Comment (sennierer) [reply docs/source/features.rst]: good catch, changed the url to the one that is actually used

**Review by b1rger — CHANGES_REQUESTED** [review]
This is now missing the `features.rst` file

**Comment by sennierer** [comment]
> This is now missing the `features.rst` file

added it now

**Review by b1rger — CHANGES_REQUESTED** [review]
I guess we can remove the `history.rst` and the `collections.rst` files in the same commit? The `features.rst` is basically a combination of those two, isn't it?

**Comment by sennierer** [comment]
yes, I will remove the files

**Review by b1rger — APPROVED** [review]

**Pull request merged**